### PR TITLE
feat(callFirestore): support GeoPoint data type

### PR DESCRIPTION
**Pull request by ytetsuro**
### Description

I tried to add a GeoPoint object from `callFirestore`.
But it was the Object that was saved.
Therefore, I modified it to convert to GeoPoint with the same logic as Timestamp.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "firebase-admin": "^8 || ^9"
   },
   "devDependencies": {
-    "@firebase/testing": "^0.20.11",
+    "@firebase/rules-unit-testing": "^1.0.5",
```
Comment (prescottprue) [inline package.json:32]: I didn't even notice when they switched this in the docs - was there a warning or error message that lead you to update this or was it just reading it in the docs?

File: test/unit/tasks.spec.ts [hunk 2]
```diff
@@ -303,6 +304,23 @@ describe('tasks', () => {
           });
         });
       });
+
+      describe('with geo point', () => {
```
Comment (prescottprue) [inline test/unit/tasks.spec.ts:308]: 👏

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,14812 @@
+{
```
Comment (prescottprue) [inline package-lock.json:1]: This can be removed since yarn is currently being used

File: package.json (hunk 1 above)
Comment (ytetsuro) [reply package.json:32]: Without it, the GeoPoint class is not generated from firesbase-admin and is considered to be a different class, so we investigated and found that `@firebase/testing` is deprecated.

**Pull request merged**